src/modules/mavlink/mavlink_main.cpp

```diff
@@ -1423,6 +1423,7 @@ Mavlink::configure_streams_to_default(const char *configure_single_stream)
 		configure_stream_local("ATTITUDE_TARGET", 2.0f);
 		configure_stream_local("AVAILABLE_MODES", 0.3f);
 		configure_stream_local("BATTERY_STATUS", 0.5f);
+		configure_stream_local("BATTERY_STATUS_V2", 0.5f);
```

Comment:
@MaEtUgR What's the normal way we handle the case where we are migrating between messages? 

My guess is that we'd initially stream both like this, but we might also update QGC to request the message type it wants and set the other not to stream?